INF-2227: Add documentation for COManage CO Group provisioning. (#144)
osg-htc/technology: docs/operations/comanage-recipes.md

COManage Recipes
==============================

A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.

> Steps for Common COManage Actions
> ----
> This section contains some common actions administrators perform on the OSG COManage, and how to do so.

Reviewer: ```suggestion
```

All of this basically just repeats what you say in the sentence above. Also, adding a section level here adds noise to the section hierarchy and it's not entirely clear why we need it (if we have another section in this doc at the same level, what would it actually be?)

### Provisioning a CO Group in COManage

> In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on Hosts),

Reviewer: ```suggestion

In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on hosts),
```

Adding a linebreak between the heading and content for readability, not sure why hosts was capitalized

>  it must first be provisioned.

Reviewer: I think it'd be worth adding an admonition here to speak to the fact that OIDC clients needing groups don't need any of these steps outside of the group creation (see https://osg-htc.org/technology/documentation/style-guide/#notes and https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types)

```
```

Follow these steps to provision a CO Group into LDAP:

> 1. #### Create CO Group in COManage and add members (or use existing CO Group)

Reviewer: ```suggestion
### Create CO Group in COManage and add members (or use existing CO Group)
```

I don't think we need to make something a section header AND an enumerated step

>     Skip the first bullet point if using an existing Group.
> 
>     -   Navigate to the `All Groups` page in COManage and click the `+ Add Group` button near the top-right.

Reviewer: ```suggestion
    1.  **If using an already existing group, skip to the next step.** Navigate to the `All Groups` page in COManage and click the `+ Add Group` button near the top-right.
```

Combining and bolding here helps with clarity. Also, this section has two steps that should happen in order (if doing both). We use bulleted lists more when we're describing exclusive options, or an unordered list

        ![Create-Group-pt1](../img/comanage-create-group.png)
        Give the group a name then click `ADD`, which will bring you to the Edit page for the new group
        ![Create-Group-pt2](../img/comanage-group-edit-page.png)
    -   Click on `MEMBERS`, then type in the name or identifier for a user you want to give membership to, 
        then select the user from the drop-down and click the `ADD` button. 
        Repeat as necessary for each group member.

>         As the creator of the Group, you will already have both Membership in, and Ownership over, the new group.

Reviewer: ```suggestion
        As the creator of the Group, you will already have both Membership in and Ownership over the new group.
        Remove yourself as appropriate.
```

>         As the creator of the Group, you will already have both Membership in, and Ownership over, the new group.

Author: ```suggestion
        As the creator of the group you will already have both Membership in, and Ownership over, the new group.
        
        Remove yourself as appropriate.
```
Github won't let me add two suggestions for the same line, so I'm just making one suggestion with both fixes (capitalization and only having one type of comma in a sentance).
